test/jdk/java/util/Formatter/BasicByteObject.java

```diff
@@ -229,7 +229,7 @@ private static void tryCatch(String fs, Class<?> ex, Object ... args) {
 
 
     private static Byte negate(Byte v) {
-        return new Byte((byte) -v.byteValue());
+        return (byte) -v.byteValue();
```

Comment:
We want to be returning a `Byte`, so casting to `(byte)` doesn't seem right to me.

`Byte.valueOf()` takes a `byte` and returns a `Byte`, so using that as the replacement for `new Byte`, we get:
`return Byte.valueOf(-v.byteValue());`
Is there a way to get the template to do that?

Reply:
So, the above suggestion gives explicit auto-boxing, though some might find it wordier than necessary.

Since `v.byteValue()` already returns a `byte`, the `(byte)` cast doesn't seem necessary. I see that L242 does:
> `return -v.intValue();`

which is a pretty clean look. So `return -v.byteValue();` would be an option, too.

Reply:
Negating v.byteValue() promotes it into an int. So then casting to byte would be necessary.

Reply:
Oh yeah...oops.